zbus/src/raw/socket.rs

```diff
@@ -40,6 +40,12 @@ fn fd_recvmsg(fd: RawFd, buffer: &mut [u8]) -> io::Result<(usize, Vec<OwnedFd>)>
 
     match recvmsg::<UnixAddr>(fd, &mut iov, Some(&mut cmsgspace), MsgFlags::empty()) {
         Ok(msg) => {
+            if msg.bytes == 0 {
+                return Err(io::Error::new(
+                    io::ErrorKind::UnexpectedEof,
```

Comment:
`io::ErrorKind::BrokenPipe` _might_ be a better choice.

Reply:
True! Thanks. BTW, let me know if you'd like to review PRs. I can really use some.